recipe/build.sh

```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
 
-if [ `uname` == Darwin ]; then
-    pythonw configure.py --sysroot=$PREFIX
-else
-    python configure.py --sysroot=$PREFIX
-fi
+python configure.py --sysroot=$PREFIX
```

Comment:
This passes... But I know nothing about `pythonw` and `OS X` so I would like a second opinion here @conda-forge/sip.